zallet/i18n/en-US/zallet.ftl

```diff
@@ -84,6 +86,12 @@ err-migrate-multiple-related-zcashd-options =
     Remove one of the conflicting options, then re-run this command.
 err-migrate-unknown-zcashd-option = Unknown {-zcashd} option '{$option}'
 
+# errors in migration of wallet data from the zcashd `wallet.dat` database format
+
+err-failed-seed-fingerprinting =
```

Comment:
I generally try to pick IDs that are namespaced:
```suggestion
err-migrate-wallet-invalid-seed =
```

Reply:
In this case, while the error will often be generated from migration, the code that it lives in isn't strictly speaking migration code, so I didn't add that prefix as I have for other migration-related error messages.